docs/standard/serverless-architecture/architecture-approaches/architecture-deployment-approaches.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Serverless apps. Architecture, patterns, and Azure implementation.
```

Comment:
title should be unique per page. It's ok to add extra keywords or the name of the e-book as well. This is what is displyed to folks when they search on Google.